File: python/stdlib/default/any.py
try:
    from ...stdlib import Node as base
except ValueError:
    from stdlib import Node as base


class Node(base.Node):
    def __init__(self, verbose, args):
        super(Node, self).__init__("Accept any input", "stdlib.default.any", "", {"val":"Object"}, {"result": "Object"},
                                   "If any input is present pass it.", verbose, loopback="val")

    def tick(self, value):
        if "tags" in value and "val" in value["tags"]:
            tag = value["tags"]["val"]
            return {"result": value["val"], "tags": {"result": tag}}
        return {"result": value["val"]}

File: lua/stdlib/default/any.lua
local myNode = {}
function myNode.init(node)
  node.tick = function(value)
    result = {}
    result.result = value.val
    return result
  end
end

function myNode.spec(node)
  node.name = "Accept any inputs"
  node.inputs.val = "Object"
  node.outputs.result = "Object"
  node.desc = "Accept any input."
end

return myNode
